config/IBAMR/packages/libmesh.py

```diff
@@ -0,0 +1,39 @@
+import config.package
+import os
+
+class Configure(config.package.GNUPackage):
+  def __init__(self, framework):
+    config.package.GNUPackage.__init__(self, framework)
+    self.download  = ['https://github.com/libMesh/libmesh/releases/download/v1.1.0/libmesh-1.1.0.tar.gz']
+    self.functions = []
+    self.includes  = ['libmesh/libmesh.h', 'libmesh/libmesh_config.h']
+    self.liblist   = [['libmesh.la']]
+    self.pkgname   = 'libmesh-1.1.0'
+    self.cxx       = 1
+    return
+
+  def setupDependencies(self, framework):
+    config.package.GNUPackage.setupDependencies(self, framework)
+    self.compilerFlags   = framework.require('config.compilerFlags', self)
+    self.mpi             = framework.require('config.packages.MPI',self)
+    self.boost           = framework.require('config.packages.boost',self)
+    self.deps            = [self.mpi, self.boost]
+    return
+
+  def formGNUConfigureArgs(self):
+    args = config.package.GNUPackage.formGNUConfigureArgs(self)
+    if self.compilerFlags.debugging:
```

Comment:
@boyceg I set the --with-methods flag based upon the compiler flags in this `if then else` block